great_expectations/rule_based_profiler/domain_builder/domain_builder.py

```diff
@@ -21,7 +20,7 @@ class DomainBuilder(ABC):
 
     def __init__(
         self,
-        data_context: DataContext,
+        data_context: "DataContext",  # noqa: F821
```

Comment:
Required to get rid of cyclic import issues - don't love the style of type annotation but it is required given our codebase dependency graph